### PR TITLE
Home section03

### PR DESCRIPTION
**Pull request by Anna-Nesterenko**
1. Fixed animation in the first section
2. Created stylization in the second and third sections
3. Launched the fourth section
4.Fixed nameRegEx validation

### CODE REVIEW

File: src/components/Home/HomeParts/HomePart1.jsx [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 const containerVariants = {
   hidden: {
     opacity: 0,
-    y: '100%',
+    // y: '100%',
   },
   visible: {
     opacity: 1,
-    y: 0,
+    // y: 0,
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart1.jsx:20]: Dont use dead code

File: src/components/Home/HomeParts/HomePart1.jsx [hunk 2]
```diff
@@ -95,8 +100,11 @@ function HomePart1() {
               <SecondText>
                 Приєднуйся до нас і працюй в команді над реальними проєктами
               </SecondText>
-              <button>Click</button>
+              <ButtonWorkWithUs title="Працювати з нами" />
             </div>
+            {/* <div>
+              <MainTitle>Jun folio</MainTitle>
+            </div> */}
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart1.jsx:107]: Dead code as well

File: src/components/Home/HomeParts/HomePart2.jsx [hunk 3]
```diff
@@ -1,47 +1,56 @@
...
-
+import {
+  Container,
+  ImagePhoto,
+  TextParagraph,
+  Section,
+} from './HomeParts.styled';
+import LogoSection from './LogoSection';
 
 function HomePart2() {
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart2.jsx:11]: semantic names for components and functions

File: src/components/Home/HomeParts/HomePart3.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import {
+  Container,
+  ImagePhoto,
+  Section,
+  TextParagraph,
+} from './HomeParts.styled';
+import LogoSection from './LogoSection';
+import { motion } from 'framer-motion';
+
+function HomePart3() {
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart3.jsx:12]: As well semantic name

File: src/components/Home/HomeParts/HomePart4.jsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Section, Container, ImageLogo } from './HomeParts.styled';
+
+function HomePart4() {
```
Comment (painlalala) [inline src/components/Home/HomeParts/HomePart4.jsx:4]: Semantic

File: src/pages/api/auth/[...nextauth].js [hunk 6]
```diff
@@ -1,8 +1,8 @@
-import User from "@component/models/User";
-import NextAuth from "next-auth";
-import CredentialsProvider from "next-auth/providers/credentials";
-import bcrypt from "bcrypt";
-import dbConnect from "@component/lib/mongodb/dbConnect";
+import User from '@component/models/User';
+import NextAuth from 'next-auth';
+import CredentialsProvider from 'next-auth/providers/credentials';
+import bcrypt from 'bcrypt';
+import dbConnect from '@component/lib/mongodb/dbConnect';
```
Comment (painlalala) [inline src/pages/api/auth/[...nextauth].js:5]: Better to split imports
1.libs
free space
2. cunstom components

**Review by painlalala — APPROVED** [review]

**Comment by painlalala** [comment]
resolve conflict and merge it)

**Review by forze961 — APPROVED** [review]

**Pull request merged**